packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -325,7 +325,27 @@ class ErrorSpacer extends DiagnosticsProperty<void> {
 
 /// Class for information provided to [FlutterExceptionHandler] callbacks.
 ///
-/// See [FlutterError.onError].
+/// This is an example of using [FlutterErrorDetails] when calling
+/// [FlutterError.reportError].
+///
+/// ```dart
```

Comment:
Can this be wrapped in a `@{tool ...}` section?

Reply:
Sure! Do you mean an app that throws an error and crashes? I can't wrap it in a `{@tool ...}` and pass the analyzer without adding a lot of irrelevant code.

Reply:
Can you just wrap this in a method to make analyzer happy?